android-auto-app/build.gradle.kts

```diff
@@ -35,7 +35,6 @@ android {
     if (buildFromSource.toBoolean()) {
       jniLibs.pickFirsts.add("**/libc++_shared.so")
```

Comment:
@ank27 do we need that by the way? I was under an impression that gl-native does not contain libc++_shared and only common does and in that case we don't need pick first

Reply:
Technically we don't need this as well. But I guess we can keep it the same as before. 
Also, I don't know if any other project builds `libc++_shared.so` file, why it was added in the first place?

Reply:
This is used for building from source, where I think gl-native indeed will bundle the `libc++_shared.so` when built from source, we could verify it by building the project from source.